kmir/src/kmir/__main__.py

```diff
@@ -60,6 +56,9 @@ def _arg_parser() -> ArgumentParser:
     run_parser = command_parser.add_parser('run', help='run stable MIR programs')
     run_parser.add_argument('input_file', metavar='FILE', help='MIR program to run')
     run_parser.add_argument('--depth', type=int, metavar='DEPTH', help='Depth to execute')
+    run_parser.add_argument(
+        '--start-symbol', type=str, metavar='SYMBOL', default='main', help='Symbol name to begin execution from'
+    )
```

Comment:
:+1: good idea.
I was struggling to find a way to have a _default value_ for a K config variable (there is probably none), this is much easier.